lib/elixir/test/elixir/macro_test.exs

```diff
@@ -674,6 +674,11 @@ defmodule MacroTest do
            [1, 2, 3, [1, 2, 3], 4, 5, 6, [4, 5, 6], {[1, 2, 3], [4, 5, 6]}]
   end
 
+  test "generate_arguments" do
```

Comment:
We usually name the tests after the exact function (so function name + arity), so this one I would call `test "generate_arguments/2"` :).